foreman-proxy-content-enable

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+set +e
+
+TMP=$(mktemp -t katello-selinux-enable.XXXXXXXXXX)
+trap "rm -rf '$TMP'" EXIT INT TERM
+LOG=/var/log/foreman-selinux-install.log
+
+for selinuxvariant in targeted
+do
+  if /usr/sbin/semodule -s $selinuxvariant -l >/dev/null; then
+    # Load policy
+    /usr/sbin/semanage module -S $selinuxvariant \
+      -a /usr/share/selinux/${selinuxvariant}/foreman-proxy-content.pp.bz2
+
+    # Append to log file
+    echo "$(date) $0" >> $LOG
+    cat $TMP_EXEC_BEFORE >> $LOG
+    cat $TMP_EXEC_AFTER >> $LOG
```

Comment:
Is this needed? It's not like we really add anything to those logs. They're not even defined.

In https://github.com/theforeman/foreman-selinux/blob/develop/foreman-selinux-enable they are actually used, but I think it's safe to drop these lines.
```suggestion
```